src/offline/ledgerhash.ts

```diff
@@ -113,7 +113,7 @@ function computeLedgerHeaderHash(
   options: ComputeLedgerHeaderHashOptions = {}
 ): string {
   const subhashes = {
-    transactionHash: computeTransactionHash(ledger, options),
+    transactionHash: computeSignedTransactionHash(ledger, options),
```

Comment:
Why are options being passed in here?

Reply:
This function happens to have the exact same name as the one that was being exported, but it's serving a slightly different function. (It gets the Transaction Hash from a Ledger object instead of from a Transaction object and also performs some additional verifications to ensure the Ledger object is properly formed)

I renamed it to be more descriptive (To `computeTransactionHashFromLedger`)